src/blog/2024/10/dashboard-new-group-type-app-icon-and-charts.md

```diff
@@ -0,0 +1,60 @@
+---
+title: New Chart Type, Customizable Icon and Groups as Dialogs Now Available in FlowFuse Dashboard
+subtitle: Our latest update for FlowFuse Dashboard introduces a new group type, Dialog, a new chart variation, Histogram and customization support for the application icon.
+description: Our latest update for FlowFuse Dashboard introduces a new group type, Dialog, a new chart variation, Histogram and customization support for the application icon.
+date: 2024-10-11
+authors: ["gayan-sandamal"]
+image: /blog/2024/08/images/dashboard-2-1150-release.png
+tags:
+   - posts
+   - news
+   - flowfuse
+   - dashboard
+---
+
+At FlowFuse, we strive to improve your dashboard experience by introducing features that enhance data visualization and customization. This update brings new ways to tailor your dashboards with key improvements, including a new chart type, a customizable app icon, and a dialog feature for groups.
+
+<!--more-->
+
+## New Chart Type: Histogram
```

Comment:
Also add a paragraph in this section that sells the value of the Histogram chart type. It will do the grouping and frequency tracking for you. You can just pass the raw data, and it'll automatically put them into categories/bins and display the relevant frequency data.

Reply:
Added a paragraph that sells the value of Histogram

"The real advantage of this new Histogram chart type is that it simplifies the process for you. Just pass in the raw data, and the histogram will automatically organize it into meaningful ranges, then display how often each range occurs. This makes it incredibly easy to extract valuable insights without the need for manual data processing."